homeassistant/components/homeassistant_yellow/manifest.json

```diff
@@ -6,5 +6,6 @@
   "config_flow": false,
   "dependencies": ["hardware", "homeassistant_hardware"],
   "documentation": "https://www.home-assistant.io/integrations/homeassistant_yellow",
-  "integration_type": "hardware"
+  "integration_type": "hardware",
+  "requirements": ["gpiod==1.5.4", "pyusb==1.2.1"]
```

Comment:
This is a rather old version of `gpiod`, was that intentional?

Reply:
`universal-silabs-flasher` currently depends on `gpiod==1.5.4`. We'd have to bump it first.

---

Back in December, `gpiod>=2` was [a major packaging issue](https://github.com/NabuCasa/universal-silabs-flasher/pull/50#issuecomment-1864821418) because it could only be built from source and required a full dev env and extra packages.

2.2.0 published in June is the first release to have binary wheels. I'll see if this is feasible now.

Reply:
This dependency upgrade has been unblocked by #122812 so we're now on the latest release of `gpiod`.